backend/src/controllers/auth.ts

```diff
@@ -6,7 +6,7 @@ import { getUserByEmail } from '../services/users'
 // handle login request
 export const login = async (req: Request, res: Response) => {
     // Get the ID token passed.
-    const idToken = req.body.idToken.toString()
+    const idToken = req.body.idToken?.toString()
     if (!idToken) {
```

Comment:
I think we should also check if the id token is expired

Reply:
We authenticate routes based on sessionCookie not idToken. So it doesn't matter because sessionCookie has an expiration time of 5 days.

Reply:
But as an additional check, yes we can add it.